sklearn/linear_model/_ridge.py

```diff
@@ -1693,6 +1695,9 @@ class RidgeCV(MultiOutputMixin, RegressorMixin, _BaseRidgeCV):
     alpha_ : float
         Estimated regularization parameter.
 
+    best_score_ : float
+        Score of base estimator with best alpha on the hold out data.
```

Comment:
the version from GridSearchCV seems better: "Mean cross-validated score of the best_estimator"

Reply:
I changed the docstring. However, I don't refer to `best_estimator` because we don't have such attributes (while it is available in GridSearchCV).